test/cat.jl

```diff
@@ -150,24 +149,24 @@ module TestCat
         dt = vcat(DataTable([[1]], [:x]), DataTable([["1"]], [:x]))
         @test dt == DataTable([[1, "1"]], [:x])
         @test typeof.(dt.columns) == [Vector{Any}]
-        dt = vcat(DataTable([NullableArray([1])], [:x]), DataTable([[1]], [:x]))
```

Comment:
Same here (`Vector{Int}` is tested above). We're going to have to be very careful when doing this, as it's very easy to lose coverage and stop testing type combinations we care about. That's one of the problems of no longer converting columns to nullable arrays by default. I won't leave a note everywhere to keep the review manageable, but please give these another pass.

In particular, there's no point in calling `Vector` on a literal which is already a vector (or even a range) before passing it to a constructor, so these cases may indicate that something is going on. OTC, you can very often remove `NullableArray` when it's used just to get `isequal` to work.

Reply:
This remark applies to the whole file.